frontend/src/components/Landing/__snapshots__/signin-button.test.js.snap

```diff
@@ -0,0 +1,12 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
why the link?

Reply:
It was auto-generated by Jest Snapshot when it made the file so I think better to just leave it alone? I don't think the snapshot files are really meant to be inspected by people anyways.